doc/sphinx-guides/source/installation/config.rst

```diff
@@ -4316,6 +4316,18 @@ can enter for an embargo end date. This limit will be enforced in the popup dial
 
 ``curl -X PUT -d 24 http://localhost:8080/api/admin/settings/:MaxEmbargoDurationInMonths``
 
+.. _:MinRetentionDurationInMonths:
+
+:MinRetentionDurationInMonths
++++++++++++++++++++++++++++++
+
+This setting controls whether retentions are allowed in a Dataverse instance and can limit the minimum duration users are allowed to specify. A value of 0 months or non-existent
+setting indicates retentions are not supported. A value of -1 allows retentions of any length. Any other value indicates the minimum number of months (from the current date) a user
+can enter for a retention end date. This limit will be enforced in the popup dialog in which users enter the retention date. For example, to set a ten year minimum:
```

Comment:
Perhaps we should mention the 1000 year hardcoded max? (Same for embargo?)

Should we note that when also allowing embargoes and allowing embargoes longer than the min retention period, files would never be available? (Real question - it's true but not sure it needs to be stated.)

Reply:
That 1000 years is a hardcoded default value, I don't think that it makes sense to mention it. 
The possibility that you have embargo and retention setup in a way that make the file never accessible for public seems unlikely. We could mention that there can also be an embargo and you have to check that the retention expires after that, but I think people will understand this.